### DISCUSSION TITLE
Rename `master` -> `main`.

### DISCUSSION THREAD

**Discussion by ioquatix** [kickstarter/rack-attack#571] [General]
Hello @grzuy we are for consistency renaming `master` to `main` branch (done for most other `rack` org repos). Please consider making that change on this repo. Kind regards, @ioquatix.

**Comment by grzuy** [comment]
Hi @ioquatix,

Sounds good to me :-)
I already pushed new `main` branch.

What would be pending is setting `main` as the repo default branch, but I don't have permissions to do so.
Happy to do it if I get permissions for that or if someone else can take care of that.

Thanks!

**Reply by ioquatix** [reply]
I can take care of it :)

**Reply by ioquatix** [reply]
Okay, I made a bit of a mess but it's all done and tidied up now.

**Reply by grzuy** [reply]
Hehe, no worries.
Thank you @ioquatix !
